### DISCUSSION TITLE
conditional sprites seem to only work in singleplayer (slugcat)

### DISCUSSION THREAD

**Discussion by kleiner-sentry** [FakeFishGames/Barotrauma#13619] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

I made a custom character with conditional sprites set to change a sprite if a condition set to change the sprite when I kill myself (among other conditional sprites). This works in singleplayer, however when I host a game in multiplayer the conditional sprite doesn't work. This is strange, because the hammer head matriarch works perfectly fine. Im not sure if my character is just cursed, or if it has something to do with the game.

[Slugcat.zip](https://github.com/FakeFishGames/Barotrauma/files/14156161/Slugcat.zip)

### Reproduction steps

1. Start a singleplayer game
2. spawn the custom character with a conditional sprite with the condition to change the sprite if you die (ideally human-like)
3. kys and observe the changed sprite
4. go to multiplayer
5. spawn the same character
6. kys and observe

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I gave this a test, but unfortunately couldn't spot any issues. The conditional sprites seem to work with no issues at my end:

https://github.com/FakeFishGames/Barotrauma/assets/14351070/86312792-037a-4d87-bddb-3c382cce7a60

https://github.com/FakeFishGames/Barotrauma/assets/14351070/11a4545f-48dd-4c3f-b5cc-2930ddd23bf8

I wonder if this could have something to do with how you're killing the character? Did you use some kind of console commands or do it "manually" somehow?

**Comment by kleiner-sentry** [comment]
Strange. It all seems to work now. But I still don't see them looking stunned when I close the door on them, like how it works in singleplayer.
